enforcer_interface.go

```diff
@@ -51,6 +52,7 @@ type IEnforcer interface {
 	EnableEnforce(enable bool)
 	EnableLog(enable bool)
 	EnableAutoNotifyWatcher(enable bool)
+	EnableautoNotifyDispatcher(enable bool)
```

Comment:
modify to `EnableAutoNotifyDispatcher`, same question on beta branch.

Reply:
I will fix this typo after this PR is merged